packages/components/src/navigator/stories/index.js

```diff
@@ -34,6 +49,27 @@ const MyNavigation = () => (
 				Go back
 			</NavigatorButton>
 		</NavigatorScreen>
+		<NavigatorScreen path="/overflow-child">
+			<NavigatorButton isPrimary path="/" isBack>
+				Go back
+			</NavigatorButton>
+			<div
+				style={ {
+					display: 'inline-block',
+					background: 'papayawhip',
+				} }
+			>
+				<span
+					style={ {
+						color: 'palevioletred',
+						'white-space': 'nowrap',
+						'font-size': '42vw',
```

Comment:
Doesn't really matter, but let's just use camelCase for these properties so we don't get console warnings.

Reply:
Thanks for that! ☑️